test/run-drun/ok/text-lowercase.wasm-run.ok

```diff
@@ -0,0 +1,13 @@
+ABCDCEFGHIJABCDCEFGHIJABCDCEFGHIJABCDCEFGHIJABCDCEFGHIJ 1234 X
```

Comment:
@crusso I just ran the tests locally and they work, and have the updated output for your updated tests. 🎉 

Did you want me or someone to check those in, or were you planning to do that?

Reply:
Oh thanks, I though I had already but maybe I forgot to push. Lemme check before you push